src/ert/validation/proper_name_argument.py

```diff
@@ -36,3 +42,27 @@ def validate(self, token: str) -> ValidationStatus:
             validation_status.setValue(token)
 
         return validation_status
+
+
+class ExperimentValidation(ProperNameArgument):
```

Comment:
This is mostly moved? Not a big deal, but could put that in a different commit to improve readability.